### PR TITLE
test: removed tests that were testing for files that didnt exist, bumped gradle version to 4.10.2

### PR DESCRIPTION
**Pull request by Strum355**
# Fixes # 

### Checklist
- [x] I have read the [Contribution Guide] and my PR follows them.
- [x] I have made a material change to the repo (functionality, testing, spelling, grammar)
- [x] I updated my branch with the master branch.
- [ ] ~~I have added tests that prove my fix is effective or that my feature works~~ No new tests needed
- [ ] ~~I have added necessary documentation about the functionality in the appropriate .md file~~ No new code added
- [ ] ~~I have added in line documentation to the code I modified~~ No new code added

### Short description of what this PR does:
Updated gradle version from 3.3 to 4.10.2
Tiny cleanup in Client.java
Removed tests that were testing for files that didn't exist

### CODE REVIEW

**Comment by SendGridDX** [comment]
[CLA assistant check](https://cla.sendgrid.com/sendgrid/java-http-client?pullRequest=93) <br/>All committers have signed the CLA.

File: src/test/java/com/sendgrid/TestRequiredFilesExist.java [hunk 1]
```diff
@@ -13,18 +13,6 @@
         assertTrue(dockerExists);
     }
 
-    // ./docker-compose.yml or ./docker/docker-compose.yml
```
Comment (thinkingserious) [inline src/test/java/com/sendgrid/TestRequiredFilesExist.java:16]: Can you please just comment out these tests for now? Eventually we want them to pass. Thanks!

**Review by thinkingserious — CHANGES_REQUESTED** [review]

File: src/test/java/com/sendgrid/TestRequiredFilesExist.java (hunk 1 above)
Comment (Strum355) [reply src/test/java/com/sendgrid/TestRequiredFilesExist.java:16]: No problem, changes are commited :+1:

Comment (Strum355) [reply src/test/java/com/sendgrid/TestRequiredFilesExist.java:16]: Actually, should i only re-add the docker-compose ones or USE-CASES.md and .env-sample as well?

**Review by thinkingserious — APPROVED** [review]

**Comment by thinkingserious** [comment]
Hello @Strum355,<br /><br/>Thanks again for the PR!<br />                                              <br />It's <a href="https://sendgrid.com/blog/hacktoberfest-2018-has-arrived/">HACKTOBERFEST</a>! We want to show our appreciation by sending                                              you some special Hacktoberfest swag. If you have not already, could you please fill out                                              <a href="https://dx.sendgrid.com/hacktoberfest/join">this form</a> so we can send it to you? Thanks!                                             <br /><br />Team SendGrid DX

**Pull request merged**